webapp/src/plugin.jsx

```diff
@@ -0,0 +1,33 @@
+import {configurationChange, fetchPluginConfiguration} from 'actions/config';
+import {websocketHasVoted} from 'actions/vote';
+
+import {id as pluginId} from './manifest';
+import reducer from './reducers';
+
+export default class MatterPollPlugin {
+    async initialize(registry, store) {
+        const data = await fetchPluginConfiguration()();
```

Comment:
The problem with this is, that if a user is not logged in, the code still tries to fetch the data, which results in a `4ß3`. Would be great to fix this, but it can be done in a different PR. I don't no how much work this is.

Reply:
Maybe https://github.com/mattermost/mattermost-plugin-nps/pull/50 helps?

Reply:
Thanks for pointing. I'll take a look into it.

Reply:
This problem could be resolved by [this commit](https://github.com/matterpoll/matterpoll/pull/160/commits/1b4cb034b5e9059a68c6ce9cb418f940586da019).

Although we can avoid the error by checking currentUserId, it will not fetch the configuration when logging in. So I think that we need to subscribe the websocket event that is published when logging in (it might be called `hello` event)